core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11971,4 +11971,16 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
 
+
+  @Test public void testShiftLeft() {
```

Comment:
Please add one more test case in ReltoSqlConverterTest class

Reply:
done